lib/datadog/kit/events.rb

```diff
@@ -7,41 +7,63 @@ module Datadog
   module Kit
     # Tracking events
     module Events
-      # Attach login event information to the trace
+      # Attach login success event information to the trace
+      #
+      # This method is experimental and may change in the future.
       #
       # @param trace [TraceOperation] Trace to attach data to.
-      # @param success [bool] Whether login operation was successful or not.
       # @param user [Hash<Symbol, String>] User information to pass to
       #   Datadog::Kit::Identity.set_user. Must contain at least :id as key.
       # @param others [Hash<String || Symbol, String>] Additional free-form
       #   event information to attach to the trace.
-      def self.track_login(trace, success:, user:, **others)
-        subtag = success ? 'success' : 'failure'
-        event = "users.login.#{subtag}"
+      def self.track_login_success(trace, user:, **others)
+        event = 'users.login.success'
```

Comment:
Should we extract this into a constant? Same for the `'users.login.failure'` string

Reply:
Good idea